static/openapi/vwrs-spec.json

```diff
@@ -1476,7 +996,7 @@
           {
             "name": "key_id",
             "in": "path",
-            "description": "Unique key_id to fetch the API key.",
+            "description": "Unique ID of the key to be fetched.",
```

Comment:
@diana-macrometa ... 
"to be fetched" is strange.  If you feel this could be improved, please provide an alternative.

Reply:
Yeah, I always forget that passive voice is bad style in English. I'll come up with something better.